metrotranscom/doorway
# fix: seed data county

# Pull Request Template

## Issue Overview

This PR addresses #issue

- [x] This change addresses the issue in full - **Limitation: links to external individual listings do not include county in the view - we've requested that Exygy update the view**

## Description
**UPDATE:** realized the spaces were the issue & added them back in throughout. Also changed/ensured that a county is included for any `buildingAddress` and that it was from the 9 possible county choices (ie changed San Jose to Santa Clara).

What it looked like on the Partners site
![image](https://github.com/metrotranscom/doorway/assets/12400715/2b1462ee-410c-455e-a541-958b64295acd)

Public site individual listings
![image](https://github.com/metrotranscom/doorway/assets/12400715/126e0cd2-05da-4c79-b92d-14bd94088189)


## How Can This Be Tested/Reviewed?
Reseed with external data `db:reseed:with-external:dev` and run both Partners & Public site and view various seed data listings to ensure that the counties are shown correctly. Check both internal & externally seeded data.  (Doorway seed data: Triton 2 is San Mateo & Coliseum is Alameda and others show San Francisco)

Should look like:
Partners site: ![image](https://github.com/metrotranscom/doorway/assets/12400715/7549f520-28a7-4d72-aa59-04e7756ae13b)

Public site Listings: 
![image](https://github.com/metrotranscom/doorway/assets/12400715/4f46d70e-2717-4f0b-98f1-bcbb013d08c0)

Public site individual listing:
![image](https://github.com/metrotranscom/doorway/assets/12400715/884aaef7-f2b8-4267-b5fb-e40cda6331cc)


## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
